### ISSUE TITLE
tour: error: taking address of temporary

### ISSUE THREAD

**Issue by adiog** [golang/tour#311]
Context: https://tour.golang.org/moretypes/5

For someone coming from C++: the following statement
```
p  = &Vertex{1, 2} // has type *Vertex
```
deserves at least a note. I am not yet familiar with go memory model, but I am still finding this disturbing. Please warn the reader that it might be more complicated than it looks like.

**Comment by adiog-ha** [comment]
Does anyone take care of issues raised? Or trying to improve your work is completely useless?

**Comment by ALTree** [comment]
> deserves at least a note.

You're not wrong, but the problem is that a note will confuse everyone that doesn't come from a C/C++ background. The tour is not really the place to talk about Go's memory model.
